framework/src/utils/MooseAppCoordTransform.C

```diff
@@ -115,16 +117,17 @@ MooseCoordTransform::setRotation(const Real alpha, const Real beta, const Real g
     }
   }
 
+  _euler_angles = {{alpha, beta, gamma}};
   _rotate = std::make_unique<RealTensorValue>(
       RealTensorValue::extrinsic_rotation_matrix(alpha, beta, gamma));
   computeRS();
 
   if (must_rotate_axes && !axes_rotated)
-    mooseError("Unsupported manual angle prescription in 'MooseCoordTransform::setRotation'");
+    mooseError("Unsupported manual angle prescription in 'MooseAppCoordTransform::setRotation'");
```

Comment:
```suggestion
    mooseError("Unsupported manual angle prescription in 'MooseAppCoordTransform::setRotation.' "
                        "Supported rotation is (alpha, beta, gamma) = (0, 90, 0)");
```

Reply:
well I wasnt adding just a point there?

Reply:
Ah indeed not

Reply:
Now I understand why that space was there!